pkg/mkpersist/build.yml

```diff
@@ -0,0 +1,18 @@
+# linuxkit build template 
+#
+# Copyright (c) 2018 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+org: lfedge
+image: eve-mkpersist
+network: yes
+config:
```

Comment:
I actually don't think you need all of this for the type of container we have in mind. Look at what mountie does: https://github.com/linuxkit/linuxkit/blob/master/pkg/mount/build.yml

Reply:
Are you referring to the bind locations? I'll remove /etc/resolv.conf and /run. Is there something else that I'm missing?
I'll change the capabilities part also to reflect what mountie has. Will remove "pid: host" part also. Anything else?